.travis.yml

```diff
@@ -220,34 +217,40 @@ before_install:
   - tools/ci/prep-travis-forssh.sh
   # Install grunt-cli
   - eatmydata npm install grunt-cli
-  # Install optionally upstream current development so we are sure that they break nothing important for us
-  - if [ ! -z "${_DL_UPSTREAM_GITANNEX:-}" ]; then sudo tools/ci/install-annex-snapshot.sh; sudo ln -s `find /usr/local/lib/git-annex.linux -maxdepth 1 -type f -perm /+x` /usr/local/bin/; else sudo eatmydata apt-get install git-annex-standalone ; fi
-  # Install optionally -devel version of annex, and if goes wrong (we have most recent), exit right away
-  - if [ ! -z "${_DL_DEVEL_ANNEX:-}" ]; then tools/ci/prep-travis-devel-annex.sh || { ex="$?"; if [ "$ex" -eq 99 ]; then exit 0; else exit "$ex"; fi; }; fi
-  - if [ ! -z "${_DL_DEBURL_ANNEX:-}" ]; then wget -O /tmp/git-annex.deb "${_DL_DEBURL_ANNEX}" && sudo dpkg -i /tmp/git-annex.deb; fi
-  # Optionally install the latest Git.  Exit code 100 indicates that bundled is same as the latest.
-  - if [ ! -z "${_DL_UPSTREAM_GIT:-}" ]; then
-      sudo tools/ci/install-latest-git.sh || { [ $? -eq 100 ] && exit 0; } || exit 1;
-    fi
-
-install:
-  # Install standalone build of git-annex for the recent enough version
+  # Install various basic depedencies
   - travis_retry sudo eatmydata apt-get install zip pandoc p7zip-full
   # needed for tests of patool compression fall-back solution
   - travis_retry sudo eatmydata apt-get install xz-utils
   - travis_retry sudo eatmydata apt-get install shunit2
   # for metadata support
   - travis_retry sudo eatmydata apt-get install exempi
+  # Configure TMPDIR before trying install git-annex -- script might use it
+  - if [[ "${TMPDIR:-}" =~ .*/sym\ link ]]; then echo "Symlinking $TMPDIR"; ln -s /tmp "$TMPDIR"; fi
+  - if [[ "${TMPDIR:-}" =~ .*/d\ i\ r ]]; then echo "mkdir $TMPDIR"; mkdir -p "$TMPDIR"; fi
+  - if [[ "${TMPDIR:-}" =~ .*/nfsmount ]]; then echo "mkdir $TMPDIR"; mkdir -p "$TMPDIR" "${TMPDIR}_"; echo "/tmp/nfsmount_ localhost(rw)" | sudo bash -c 'cat - > /etc/exports'; sudo apt-get install -y nfs-kernel-server; sudo exportfs -a; sudo mount -t nfs localhost:/tmp/nfsmount_ /tmp/nfsmount; fi
+  # Install git-annex
+  - OLD_PATH="$PATH"
```

Comment:
Now I am thinking that we might just want to add options (e.g. `--symlink-under=PATH`) to the install script to install `git-annex` and `git-annex-shell` into `/usr/local/bin` which we would use in such cases and avoid the need for `source` (which should still be "an option" for other testing deployments)